CHANGELOG.md

```diff
@@ -2,25 +2,27 @@
 
 ## Unreleased
 
+## 3.14.0 (Feature release)
+
 ### New features
 
-#### Use override classes to set text alignment
+#### Set text alignment with override classes
 
 You can now use the `govuk-!-text-align-left`, `govuk-!-text-align-centre` and `govuk-!-text-align-right` CSS classes to set text alignment on elements.
 
-This was added in [pull request #2339: Add text align override classes](https://github.com/alphagov/govuk-frontend/pull/2339).
+This was added in [pull request #2339: Add `text-align` override classes](https://github.com/alphagov/govuk-frontend/pull/2339). Thanks to [Ed Horsford](https://github.com/edwardhorsford) for reporting this issue.
 
 #### Define negative spacing using the `govuk-spacing` function
 
 You can now pass the negative equivalent of a point from the spacing scale to the `govuk-spacing` function to get negative spacing.
 
 For example, `govuk-spacing(1)` returns `5px`, and `govuk-spacing(-1)` returns `-5px`.
 
-This was added in [pull request #2348: Allow govuk-spacing to output negative spacing](https://github.com/alphagov/govuk-frontend/pull/2348).
+This was added in [pull request #2348: Allow `govuk-spacing` to output negative spacing](https://github.com/alphagov/govuk-frontend/pull/2348). Thanks to [Chris Hill-Scott](https://github.com/quis) for reporting this issue.
 
 ### Fixes
 
-- [#2347: Fix panel text overflowing when zoomed in on mobile #2347](https://github.com/alphagov/govuk-frontend/pull/2347)
+- [#2347: Prevent panel text overflowing when zoomed in on mobile](https://github.com/alphagov/govuk-frontend/pull/2347) - thanks to [Phil Sherry](https://github.com/philsherry) for reporting this issue.
```

Comment:
How come this doesn't follow the same format as lines 13 and 21 with a full stop after the link?

Reply:
I think it's to do with how we mention the pull request...

Lines 13 and 21 are paragraphs, and both have a sentence like "This was added in pull request #1234."

Whereas, line 25 is just a bullet, so I didn't think to put a full stop after the link.

Reply:
ah that makes sense, thanks. resolving this comment

Reply:
Thanks for your help :)